SH4RE/SH4RE/ListingViewModel.swift

```diff
@@ -176,7 +179,7 @@ class ListingViewModel : ObservableObject{
                                           }
                                       }
                                       if(available == true){
-                                          let listing = Listing(id:hit.objectID, uid: hit.UID, title: hit.Title, description: hit.Description, imagepath : hit.image_path, price: hit.Price, availability : hit.Availability, category: hit.Category, address: hit._geoloc, ownerName : hit.ownerName, timestamp:hit.timestamp)
+                                          let listing = Listing(id:hit.objectID, uid: hit.UID, title: hit.Title, description: hit.Description, imagepath : hit.image_path, price: hit.Price, availability : hit.Availability, category: hit.Category, address: hit._geoloc, ownerName : hit.ownerName, timestamp:hit.timestamp, postalCode: hit.postal)
```

Comment:
wait why do we have address AND postal code now?

Reply:
We need postal code so we can prepopulate the location field when the user goes to edit listing. I thought we could get rid of this but didn't realize it was used here so I added back